### ISSUE TITLE
Creating AWS SES Configuration Set sometimes fails in Terraform 0.12

### ISSUE THREAD

**Issue by zipbee** [hashicorp/terraform-provider-aws#8940]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

`Terraform v0.12.1`

### Affected Resource(s)

* aws_ses_configuration_set

### Terraform Configuration Files

```hcl
provider "aws" {
  alias  = "eu-west-1"
  region = "eu-west-1"
}

resource "aws_ses_configuration_set" "ses_config_set" {
  name     = "client-${var.client_id}-${var.env}"
  provider = aws.eu-west-1
}
```

### Debug Output

Snippets of the full debug output: https://gist.github.com/zipbee/19f2fcd8efc40e79901d50ab3e17f61b
It looks like this happens:
* Request to create ConfigurationSet
* 200 OK response from creation request
* Request to list all configuration sets
* 200 OK response from list request
* Newly created ConfigurationSet is not listed in the output of the list command

This might be a timing issue, e.g. AWS creates it but doesn't instantly list it when requesting a list of all configuration sets?

### Expected Behavior

Resource should be created and successfully tracked.

### Actual Behavior
Apply fails with:
```
Error: Provider produced inconsistent result after apply

When applying changes to aws_ses_configuration_set.ses_config_set, provider
"aws" produced an unexpected new value for was present, but now absent.

This is a bug in the provider, which should be reported in the provider's own
issue tracker.
```

Re-running the Apply fails because a the resource already exists.

### Steps to Reproduce

1. `terraform apply`

### Important Factoids

We use this in various projects and it sometimes works and sometimes doesn't, everything has been upgraded to Terraform v0.12 and we can't find a clear indicator for what breaks it. We never saw this issue in Terraform 0.11 at least.

**Comment by zipbee** [comment]
We've been trying to narrow down what triggers this issue as we have a lot of instances where this works successfully. So far we can only conclude it is down to the name of the configuration set but can't discern a pattern.

We have some examples that work:
* client-exa-dev
* client-goz-dev
* client-nstest8-dev

And some that definitely don't work:
* client-zog-dev (goz backwards)
* client-zogzog-dev (length doesn't matter)
* client-zig-dev (different letter combinations)
* client-sig-dev
* client-zic-dev
* client-sic-dev

Here's an example output of a successful creation for "client-goz-dev":
```
Terraform will perform the following actions:

  # aws_ses_configuration_set.ses_config_set will be created
  + resource "aws_ses_configuration_set" "ses_config_set" {
      + id   = (known after apply)
      + name = "client-goz-dev"
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

aws_ses_configuration_set.ses_config_set: Creating...
aws_ses_configuration_set.ses_config_set: Creation complete after 0s [id=client-goz-dev]

Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
```

And the exact same project with the name changed to "client-zog-dev":
```
Terraform will perform the following actions:

  # aws_ses_configuration_set.ses_config_set will be created
  + resource "aws_ses_configuration_set" "ses_config_set" {
      + id   = (known after apply)
      + name = "client-zog-dev"
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

aws_ses_configuration_set.ses_config_set: Creating...

Error: Provider produced inconsistent result after apply

When applying changes to aws_ses_configuration_set.ses_config_set, provider
"aws" produced an unexpected new value for was present, but now absent.

This is a bug in the provider, which should be reported in the provider's own
issue tracker.
```

Feels like we're going mad! It would be useful if someone else can test this and confirm if it works for them or not.

**Comment by anGie44** [comment]
A potential fix has been merged and is set to release with `v2.65.0 ` of the Terraform AWS Provider, to be out later this week.
